elixir_phoenix/web/controllers/client_controller.ex

```diff
@@ -0,0 +1,58 @@
+defmodule ElixirPhoenix.ClientController do
+  use ElixirPhoenix.Web, :controller
+
+  alias EboshiApiShootoutElixirPhoenix.Client
+  alias EboshiApiShootoutElixirPhoenix.Repo
+
+  plug :scrub_params, "client" when action in [:create, :update]
+
+  def index(conn, _params) do
+    clients = Repo.all(Client)
+    render(conn, "index.json", clients: clients)
+  end
+
+  def create(conn, %{"client" => client_params}) do
```

Comment:
this file is mostly scaffold.  i absolutely don't need anything other than `index` to pass current test, but leaving it here in anticipation of future test cases for POST/PUT/etc.   I'll probably tend to commit some endpoints beyond what is strictly needed for test passes just so i can play with the language and Phoenix.